app/templates/components/navbar.html

```diff
@@ -7,6 +7,17 @@
         <li>
           <a id="paste-btn">Paste</a>
         </li>
+        <li>
+          <a>Cursor<span class="caret"></span></a>
+          <ul id="cursor-list">
+            <li class="dropdown-item"><a href="#" onclick="return setCursor(1);">Default</a></li>
+            <li class="dropdown-item"><a href="#" onclick="return setCursor(2);">None</a></li>
+            <li class="dropdown-item"><a href="#" onclick="return setCursor(3);">Crosshair</a></li>
+            <li class="dropdown-item"><a href="#" onclick="return setCursor(4);">Dot</a></li>
+            <li class="dropdown-item"><a href="#" onclick="return setCursor(5);">Pointer</a></li>
+            <li class="dropdown-item"><a href="#" onclick="return setCursor(6);">Cell</a></li>
```

Comment:
I'd prefer to avoid the magic numbers and coupling here. The hardcoded numbers mean that if we reorder the list of cursors in `setCursor`, then this navbar will break.

I created a PR with suggested rewrites that avoid the coupling:

https://github.com/djclueless/tinypilot/pull/2

If you merge it into your branch before rebasing onto master, it should go in cleanly.

Reply:
I agree, but this was the easiest way to "highlight" the active cursor in the dropdown. I can use names instead of numbers and create an extra dict in the function to keep track of the list order. {"crosshair":3,"dot":4...}

I merged in your PR